schema.development.kf

```diff
@@ -107,16 +91,15 @@ table write_grants {
     foreign_key (wg_owner_user_id) references users(id) on_delete cascade
 }
 
-// internal access grants can be granted if such permission is in write_grants
-table internal_access_grants {
+table access_grants {
     id uuid primary,
     ag_owner_user_id uuid notnull,
     ag_grantee_wallet_identifier text notnull,
```

Comment:
Maybe it's time to drop the `ag_` prefix?

Reply:
Not sure. We have `ag_`, `wg_`, `dag_`

Reply:
I would prefer not to rename it, at least for now.